### PR TITLE
Contributing guidelines, conventions

### PR DESCRIPTION
**Pull request by kdmnk**
What do you think? Did I miss anything?

### CODE REVIEW

**Comment by horcsinbalint** [comment]
I think it is great, I will post some suggestions.

File: _CONTRIBUTING.md [hunk 1]
```diff
@@ -4,11 +4,20 @@
 - [Laravel](https://laravel.com/)
 - [Materialize](https://materializecss.com/)
 
+# GitHub, Git, Branching
+
+- We organise our work around issues. Issues for structural changes should be discussed beforehand.
+- We make a new branch for every issue (or connected issues).
+- We open pull requests to the "development" branch.
+- We regularly update the "deployment" branch with changes on "development".
```
Comment (horcsinbalint) [inline _CONTRIBUTING.md:13]: We could probably mention `staging`. I usually merge new changes to `staging`, test it out, and then deploy it to production by merging new changes to `deployment`

File: _CONTRIBUTING.md [hunk 2]
```diff
@@ -25,4 +34,46 @@ git push origin your_feature_branch
...
+- Use policies instead of gates.
+- Use observers instead of events or triggers.
+- Use attributes for custom getters/setters.
+
+Structure:
+
+- Utilize resources whenever possible, following the index-create-store-show-edit-update-destroy naming convention if applicable.
+- Refrain from adding logic to the User model; delegate logic to other models where possible.
+
+Translation:
```
Comment (horcsinbalint) [inline _CONTRIBUTING.md:64]: We should probably specify that translations are required in English and Hungarian. Others don't work AFAIK.

Comment (kdmnk) [reply _CONTRIBUTING.md:64]: They should work, coming from the language contributions in the database. It's another thing that we don't have much contribution.

File: _CONTRIBUTING.md (hunk 1 above)
Comment (kdmnk) [reply _CONTRIBUTING.md:13]: Do we have a staging server as well?

Comment (horcsinbalint) [reply _CONTRIBUTING.md:13]: Yep, I created a staging server a couple of weeks ago

File: _CONTRIBUTING.md (hunk 2 above)
Comment (horcsinbalint) [reply _CONTRIBUTING.md:64]: Oh, I thought there was a technical problem but now I can see that it works

**Review by sszajbely — APPROVED** [review]
Nice!

**Pull request merged**